packages/site/src/hooks/usePageTitle.ts

```diff
@@ -0,0 +1,9 @@
+import { useEffect } from "react";
+
+function usePageTitle({ title }: { title: string }) {
+  useEffect(() => {
+    document.title = title;
```

Comment:
What do you think about doing something like this here instead:

`document.title = title === "Atlantis" ? title : `${title} - Atlantis`;`

Looking at other sites, they tend to always have their brand name appended (ex [Shopify](https://polaris.shopify.com/components/actions/button), [React](https://react.dev/reference/react-dom/components/form), [Vue](https://vuejs.org/guide/quick-start.html))

Reply:
I really like the suggestion. I updated the title on the home page to Home so I don't think think we need the conditional